configuration/items.md

```diff
@@ -49,7 +49,8 @@ The example above defines an item:
 * with name `Sensor_Temperature`
```

Comment:
Most certainly. In fact now that I look at this again, "Temperature" is a horrible item name. In my home this would be called LR_Temperature. Would you come up with a PR for LivingRoom_Temperature or whatever you think is appropriate?